pkg/kubelet/status/status_manager.go

```diff
@@ -159,7 +158,7 @@ func (m *manager) Start() {
 	//lint:ignore SA1015 Ticker can link since this is only called once and doesn't handle termination.
 	syncTicker := time.Tick(syncPeriod)
 	// syncPod and syncBatch share the same go routine to avoid sync races.
-	go wait.Forever(func() {
+	go func() {
```

Comment:
Using wait.Forever can make the goroutine keep running again even if there's a panic inside the func, so we should keep it as it is

Reply:
Hmm, the `wait` package calls `runtime.HandleCrash()` which seems to actually also `panic` by default (it logs the panic though), so I *think* this doesn't prevent a full-on crash :thinking:

Reply:
It's true that by default `runtime.HandleCrash()` recovers and then panics again, but it runs the panic handlers (if any) in between. My take is that unless there is a more through analysis that this change is actually safe, and that the extra checks and safeties provided by `wait.Forever` are unneeded, I think it's better to keep the `wait.Forever` call.

Reply:
Should we then instead just add a `defer runtime.HandleCrash` to the goroutine?